app/lib/forms/question_types/base.rb

```diff
@@ -32,9 +50,9 @@ def question_text
         I18n.t("helpers.#{title_locale_type}.registration_wizard.#{name_locale_key}")
       end
 
-    private
-
-      attr_reader :style_options
+      def body=(value)
+        super Array.wrap(value)
+      end
```

Comment:
Not entirely sure reaching into ViewComponent is a good idea here.

Reply:
If I replace the method above with this the whole test suite passes:

```ruby
def body
  []
end
```

Reply:
@peteryates - Body is not a part of a ViewComponent - it's just one of the attributes defined on the base question_type. It defined a list of paragraphs to be displayed between the h1 and the question - they are not part of the tests, so forcing them to empty array should not cause any test failures (unless they explicitly test for content).